src/components/ModalPin.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { t } from 'ttag';
+import { connect } from "react-redux";
+import $ from 'jquery';
+import PinInput from './PinInput';
+import hathorLib from '@hathor/wallet-lib';
+import PropTypes from "prop-types";
+
+
+const mapStateToProps = (state) => {
+  return {
+    wallet: state.wallet,
```

Comment:
Yes, it's used to validate the Wallet Service auth token, if needed.

https://github.com/HathorNetwork/hathor-wallet/blob/6a2240b92af70b28e09223f97fdc898f551e0d20/src/components/ModalSendTx.js#L88

Reply:
I know but it's a different file. Do you need this redux state in this `ModalPin` file?

Reply:
You're right, this parameter was not necessary and it was removed on [3b6b0c1](https://github.com/HathorNetwork/hathor-wallet/pull/392/commits/3b6b0c10871c66aebcbeb27b48cf3e7c3d638a7c).
Also removed the `mapStateToProps` since it had no other uses here. Updated the tests accordingly.